segmentation/utils/spatialorgmarkerexpression_utils.py

```diff
@@ -0,0 +1,181 @@
+import pandas as pd
+import numpy as np
+import scipy
+import statsmodels
+from statsmodels.stats.multitest import multipletests
+
+# Erin's Data Inputs
+"""
+cell_array = pd.read_csv("/Users/jaiveersingh/Downloads/SpatialEn"
+                         "richment/granA_cellpheno_CS-asinh-norm_revised.csv")
+marker_thresholds = pd.read_csv("/Users/jaiveersingh/Downloads/Sp"
+                                "atialEnrichment/markerThresholds.csv")
+dist_matrix = np.asarray(pd.read_csv("/Users/jaiveersingh/Documen"
+                                     "ts/MATLAB/distancesMat5.csv",
+                                     header=None))
+"""
+
+# Test array inputs
+"""
+pv_cellarray = pd.read_csv("/Users/jaiveersingh/De
+sktop/tests/pvcelllabel.csv")
+
+pv_distmat = np.asarray(pd.read_csv("/Users/jaiveersingh/
+Desktop/tests/pvdistmat.csv", header = None))
+
+pv_cellarrayn = pd.read_csv("/Users/jaiv
+eersingh/Desktop/tests/pvcellarrayN.csv")
+
+pv_distmatn = np.asarray(pd.read_csv("/Users/jaiveers
+ingh/Desktop/tests/pvdistmatN.csv", header = None))
+
+randMat = np.random.randint(0,200,size=(60,60))
+np.fill_diagonal(randMat, 0)
+
+pv_cellarrayr = pd.read_csv("/Users/jaiveers"
+ingh/Desktop/tests/pvcellarrayR.csv")
+"""
+
+
+def helper_function(
+        patient_data, patient_data_markers, thresh_vec, dist_mat, marker_num):
```

Comment:
This is hard to read, keep it on one line. 

Change your pycharm settings to have 99 characters per line instead of 80, that's what we use